android/app/src/main/AndroidManifest.xml

```diff
@@ -36,6 +38,13 @@
                 <category android:name="android.intent.category.LAUNCHER"/>
             </intent-filter>
         </activity>
+        <receiver android:name="com.dexterous.flutterlocalnotifications.ScheduledNotificationBootReceiver">
+            <intent-filter>
+                <action android:name="android.intent.action.BOOT_COMPLETED"/>
+                <action android:name="android.intent.action.MY_PACKAGE_REPLACED"/>
```

Comment:
"...MY_PACKAGE_REPLACED"?

Reply:
i forgot to remove the entire thing. since I removed the package, it's not required anymore